sfm/ui/templates/base.html

```diff
@@ -9,6 +9,9 @@
     <meta name="author" content="">
     <link rel="shortcut icon" type="image/png" href="{% static 'ui/img/favicon.ico' %}" />
 
+    {% block title2 %}
```

Comment:
@adhithyakiran Is this block still needed? if not, we should probably delete.

Reply:
I have deleted those block, we don't need that